### ISSUE TITLE
package.json doesn't get populated with sub-path dependencies 

### ISSUE THREAD

**Issue by dennis-standpoint** [vladshcherbin/rollup-plugin-generate-package-json#4]
I just upgraded from 3.0.0 to 3.1.1 and for some reasons sub-path requirements are not recognized.

My rollup js output contains:
```
var apiRouter = _interopDefault(require('gcf-api-router'));
require('core-js/modules/es6.regexp.split');
var Sentry = require('@sentry/node');
require('core-js/modules/web.dom.iterable');
var datastore$7 = require('@google-cloud/datastore');
var UUID = _interopDefault(require('uuid'));
var sgMail = _interopDefault(require('@sendgrid/mail'));
var bigquery$2 = require('@google-cloud/bigquery');
require('core-js/modules/es6.regexp.replace');
```

and with 3.0.0 the package.json file is
```
{"dependencies":{"@google-cloud/bigquery":"^2.0.6","@google-cloud/datastore":"^3.1.1","@sendgrid/mail":"^6.3.1","@sentry/node":"^4.6.4","gcf-api-router":"^1.0.2","uuid":"^3.3.2"}}
```

however when i upgrade to 3.1.1, the package.json only have
```
{"dependencies":{"gcf-api-router":"^1.0.2","uuid":"^3.3.2"}}
```
but the js output is still the same. Do i need to add something for it to pick up the sub-path imports?

**Comment by vladshcherbin** [comment]
@dennis-standpoint hey, thanks for the report

Yeah, this is because of the last update (3.1.1). I tried to deal with subpath exports, but forgot about packages with scopes.

I'll try to fix it this weekend, stick with 3.1.0 for now please :)

**Comment by vladshcherbin** [comment]
I've released 3.1.2, update to see if it works in your project. 😉

**Comment by dennis-standpoint** [comment]
works perfectly 😄 thanks!
